src/engraving/dom/lyrics.cpp

```diff
@@ -171,6 +171,15 @@ void Lyrics::layout2(int nAbove)
     LayoutData* ldata = mutLayoutData();
     double lh = lineSpacing() * style().styleD(Sid::lyricsLineHeight);
 
+    if ((staffType() && staffType()->xmlName() == "stdJianpu")
```

Comment:
Checking for string equality every time is not the most elegant solution. (For example, this requires you to be very careful about typing "stdJianpu" correctly every time, and the compiler won't help you detecting mistakes... and checking string equality is a relatively expensive operation.)

It would be much nicer if something similar to `isTabStaff` and `isDrumStaff` could be done.